pkgpanda/docs/create_and_maintain_packages.md

```diff
@@ -0,0 +1,79 @@
+# Creating and Maintaining Packages
+
+`pkgpanda packages` are tarballs, containing (optionally) a metadata file `pkginfo.json` as well as several well-known
+directories. See [Package Concepts](package_concepts.md) for a full listing, as well as descriptions of each section.
+should be used for.
+
+Packages are always extracted and run from a predictable location. The standard prefix is
+`/opt/mesosphere/{name}-{id}/` where `{name}` is the package name and `{id}`, a id associated with the package . This
+is effectively the `PREFIX` of the installed package in the `autotools` terms. A package may rely on the absolute path
+to the package contents to remaining constant.
+
+Only one package of a given package name may be active on a system at a given time. If a package is not active, then
+there is no guarantee that the package is available on the machine, and files in it should not be referenced.
+
+Pkgpanda operates by swapping out (roughly atomically) one set of active packages for another. If the health-checks of
+critical components fail while swapping out, Pkgpanda will attempt to revert (another move of directories), and restart
+the services again.
+
+Packages are declared in the [package store](../../packages) which is implicitly organized into a tree which can be chunked into subtrees for more specialized artifact packages.
```

Comment:
changed to: `Packages are declared in the [package store](../../packages) where they can be organized into sets called [trees](tree_concepts.md)`